charts/oidc-proxy/templates/configmap.yaml

```diff
@@ -173,6 +174,12 @@ data:
             ${FILTER3}
         </Location>
 
+        # DRS GetServiceInfo endpoint can be accessed without any authentication.
+        <Location ${PROXY_PATH3}/drs/v1/service-info>
+            ${AUTH_TYPE}
+            ${AUTH_REQUIRE}
+        </Location>
```

Comment:
On my [personal dev env](https://jade-sh.datarepo-dev.broadinstitute.org/swagger-ui.html#/DataRepositoryService/GetServiceInfo), I can also access this endpoint without authorization. But, on [Dev ](https://jade.datarepo-dev.broadinstitute.org/swagger-ui.html#/DataRepositoryService/GetServiceInfo)I need to authenticate. So, I just wanted to flag that this might be a false positive that it works on your dev env -- Just something to watch out for as we check out the behavior on dev.

Reply:
Thank you for flagging this!  I am remembering now that you shared this with me when we met the other week.

For some reason, on my dev environment prior to these changes I needed to authenticate to call this endpoint, so I am hoping it's a true positive.  I'll double check behavior in dev once it's live though.

Reply:
Ok great! Then that shouldn't be an issue.